Sources/Networking/WebBillingHTTPRequestPath.swift

```diff
@@ -48,9 +48,8 @@ extension HTTPRequest.WebBillingPath: HTTPRequestPath {
 
     var relativePath: String {
         switch self {
-        case let .getWebProducts(appUserId, productIds):
-            let productIdsQuery = productIds.map(\.trimmedAndEscaped).joined(separator: "&id=")
-            return "/rcbilling/v1/subscribers/\(appUserId.trimmedAndEscaped))/products?id=\(productIdsQuery)"
+        case let .getWebProducts(appUserID):
+            return "/rcbilling/v1/subscribers/\(appUserID.trimmedAndEscaped)/offering_products"
```

Comment:
unrelated question, what's the reasoning for being `v1` or `v2`? 🤔

Reply:
Well, I believe all SDK endpoints currently point to V1... but we do have a V2 for public facing APIs mostly... In any case, it was done this way to support versioning, but we haven't really versioned this endpoints much until now (it's more painful so we've been trying to avoid it if possible)